doc/10-icinga-template-library.md

```diff
@@ -2259,6 +2259,42 @@ snmp_service_count         | **Optional.** Compare matching services with a spec
 snmp_service_showall       | **Optional.** Show all services in the output, instead of only the non-active ones. Defaults to false.
 snmp_service_noregexp      | **Optional.** Do not use regexp to match NAME in service description. Defaults to false.
 
+### elasticsearch (NETWAYS) <a id="plugin-check-command-elasticsearch"></a>
+
+This category includes all subcommands using the [check_elasticsearch](https://github.com/NETWAYS/check_elasticsearch) plugin.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                       | Description
+---------------------------|-----------------------------------------------------------------------------------------
+elasticsearch_hostname     | **Optional.** Hostname of the Elasticsearch instance. Defaults to "localhost".
+elasticsearch_port         | **Optional.** Port of the Elasticsearch instance. Defaults to 9200.
+elasticsearch_username     | **Optional.** Username if authentication is required. No value defined as default.
+elasticsearch_password     | **Optional.** Password if authentication is required. No value defined as default.
+elasticsearch_tls          | **Optional.** Use a HTTPS connection. Defaults to false.
+elasticsearch_insecure     | **Optional.** Skip the verification of the server's TLS certificate. Defaults to false.
+elasticsearch_timeout      | **Optional.** Timeout in seconds for the CheckPlugin. Defaults to 30 seconds.
```

Comment:
It's not clear that this section does not describe a check command called `elasticsearch` but rather only common options for the following actual commands called `elasticsearch-query` and `elasticsearch-health`. A section heading that does not look like a check name would definitely help and the text should also mention that, something like "there are individual check commands for the subcommands of this alternative to/fork ok(?) check_elasticsearch [...] common custom variables for all check commands in this section"